kura/distrib/src/main/resources/intel-up2-centos-7/kura_install.sh

```diff
@@ -52,15 +52,22 @@ rm /etc/resolv.conf
 cp ${INSTALL_DIR}/kura/install/recover_default_config.init ${INSTALL_DIR}/kura/bin/.recoverDefaultConfig.sh
 chmod +x ${INSTALL_DIR}/kura/bin/.recoverDefaultConfig.sh
 
-cp ${INSTALL_DIR}/kura/install/iptables.init /etc/sysconfig/iptables
-cp /etc/sysconfig/iptables ${INSTALL_DIR}/kura/.data/iptables
+#set up default firewall configuration
+cp ${INSTALL_DIR}/kura/install/iptables.init ${INSTALL_DIR}/kura/.data/iptables
+chmod 644 ${INSTALL_DIR}/kura/.data/iptables
+cp ${INSTALL_DIR}/kura/.data/iptables /etc/sysconfig/iptables
+cp ${INSTALL_DIR}/kura/install/firewall.init ${INSTALL_DIR}/kura/bin/firewall
+chmod 755 ${INSTALL_DIR}/kura/bin/firewall
```

Comment:
Do we need also the firewall.service here?

Reply:
does firewalld use iptables ?

Reply:
No, `firewalld` is a replacement of `iptables`. But the `firewall.service` is a service added by Kura that runs `/opt/eclipse/kura/bin/firewall`at startup.

Reply:
i will add it